include/nccl_ofi_tracepoint.h

```diff
@@ -80,13 +82,13 @@
 	NCCL_OFI_TRACE_RECV_END_NVTX(request); \
 } while(0)
 
-#define NCCL_OFI_TRACE_RECV_CTRL_SEND_COMPLETE(request) do { \
-	lttng_ust_tracepoint(nccl_ofi_plugin, Recv_ctrl_send_complete, request); \
+#define NCCL_OFI_TRACE_RECV_CTRL_SEND_COMPLETE(dev, request) do { \
+	lttng_ust_tracepoint(nccl_ofi_plugin, Recv_ctrl_send_complete, dev, request); \
```

Comment:
We have duplicate events (`RECV_CTRL_SEND_COMPLETE` and `SEND_CTRL_END`) that we should clean up, but can do that in a follow-up.